scripts/helper-image.sh

```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/env bash
+
+help() {
+  echo "Usage: $0 --hostname [hostname] --identity [1|2|3]"
+  echo "- hostname: public hostname that will appear on TLS certificate"
+  echo "- identity: helper identity to be fixed inside the IPA build"
+}
+
+parse_args() {
+  while [ "${1:-}" != "" ]; do
+    case "$1" in
+      --identity)
+        shift
+        identity="$1"
+        ;;
+      --hostname)
+        shift
+        hostname="$1"
+        ;;
+      *)
+      # unknown option
+      help
+      exit 1
+    esac
+    shift
+  done
+}
+
+parse_args "$@"
+if [[ -z "${identity}" || -z "${hostname}" ]]; then
+    help
+    exit 1
+fi
+
+branch="$(git rev-parse --abbrev-ref HEAD)"
+if [[ "$branch" == "main" ]]; then
+  tag="private-attribution/ipa:latest.$identity"
```

Comment:
`latest` tags can be a source of confusion -- if we have to spin multiple versions of the helper image, it's not always obvious what version somebody has. And in some environments, a locally-stored `latest` image may be used without checking that it matches the latest available in the registry. I generally prefer to use the git SHA or a simple integer version in the tag instead.

Since the helper identity is also an integer, it may get confused with a version indicator. Maybe name the images `ipa-test-h{n}`?

Reply:
sounds good to me. Branch is redundant in this case imo